### ISSUE TITLE
[Bug] v.edit option=break does not work as expected

### ISSUE THREAD

**Issue by jradinger** [OSGeo/grass#415]
Breaking a single line (using `v.edit`) into several at multiple given point locations does produce somehow unreasonable results. More specifically, when using `v.edit` with `option=break` and two coordinate pairs to break a single line: This results, if both points are provided at the same time (i.e. within the same command), in 4 categories (features?)! I'd expect that breaking a single line at two points would result in three lines.
When the line is broken consecutively, i.e. only one point coordinate pair is provided at one time, then the result is as expected 3 lines/categories.

**To Reproduce (based on the NC dataset)**
```
##########################################################
####### VERSION 1: Break lines at two points #############
####### THIS DOES NOT WORK: Results in 4 cats ############
# Extract single line for testing
v.extract --o input=streams@PERMANENT cats=92551 output=selected_stream_v1_1

# Break line at two point locations and clean (recommended after v.edit break)
v.edit map=selected_stream_v1_1 tool=break threshold=30 coords='635861,224469,635793,224553'
v.clean --o input=selected_stream_v1_1 output=selected_stream_v1_2 tool=rmdupl

# Add categories to layer 2 to check
v.category --o input=selected_stream_v1_2 layer=2 output=selected_stream_v1_3 option=add
v.db.addtable map=selected_stream_v1_3 layer=2
v.category input=selected_stream_v1_3 option=report

##########################################################
##### VERSION 2: Break lines at each point separately ####
####### THIS WORKS: Results in 3 cats ################
# Extract single line for testing
v.extract --o input=streams@PERMANENT cats=92551 output=selected_stream_v2_1

# Break line at two point locations and clean (recommended after v.edit break)
v.edit map=selected_stream_v2_1 tool=break threshold=30 coords='635861,224469'
v.edit map=selected_stream_v2_1 tool=break threshold=30 coords='635793,224553'
v.clean --o input=selected_stream_v2_1 output=selected_stream_v2_2 tool=rmdupl

# Add categories to layer 2 to check
v.category --o input=selected_stream_v2_2 layer=2 output=selected_stream_v2_3 option=add
v.db.addtable map=selected_stream_v2_3 layer=2
v.category input=selected_stream_v2_3 option=report
```

**Expected behavior**
I'd expect that breaking a single line at two points would result in three lines. This should also work when v.edit is provided multiple points/coordinate pairs at one time.

**System description:**
GRASS version: 7.9.dev                                                          
Code revision: 8eba412c8                                                        
Build date: 2020-02-27                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.1.0dev-c74609560e                                                       
PROJ: 6.2.1                                                                     
GEOS: 3.8.0                                                                     
SQLite: 3.29.0                                                                  
Python: 3.7.5                                                                   
wxPython: 4.0.6                                                                 
Platform: Linux-5.3.0-40-generic-x86_64-with-Ubuntu-19.10-eoan

**Comment by HuidaeCho** [comment]
Version 1 breaks the "same" line twice at those two points. selected_stream_v1_1 (1 line) is split into two at the first point, then again, the same selected_stream_v1_1 (1 line) is split into another two at the second point. That's why you get 4 lines. Internally,
```
for each line:
   for each point:
      split the current line at this point <--- here, the current line is NOT updated to the new split line
```
If you want three lines, version 2 is correct.

**Comment by jradinger** [comment]
I see how it works technically, but does version 1 make sense (from a user perspective)? As said I'd rather expect an outcome of 3 lines if two points are provided!? So would it make sense (1) to change the "default" behaviour, or (2) at least clarify the functioning in the manual (and provide an example of a manual loop to obtain 3 lines, or something similar)?

**Comment by HuidaeCho** [comment]
I agree. For most users, the default behavior of version 1 is confusing and not useful. I'm not sure if it was intended or an oversight.
